qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties

```diff
@@ -1,7 +1,7 @@
-webapp.url=https://qa-refapp.openmrs.org/openmrs
+webapp.url=http://localhost/openmrs
```

Comment:
isn't test.properties file settings responsible for 2.x  RefApp  workflow tests?

Reply:
Oh yes, please remove this @kumuditha-udayanga. 

Thanks @kdaud !